draft-mouris-cfrg-mastic.md

```diff
@@ -237,9 +285,98 @@ This document uses the following terms as defined in {{!VDAF}}:
+* `Field.zeros(length: int) -> list[Self]` returns a vector of zeros of size
+  `length`.
+
+The following parameters are also referenced:
+
+  | Parameter           | Description                                           | Value                   |
+  |:--------------------|:------------------------------------------------------|:------------------------|
+  | `field: type[F]`    | class object for the field ({{Section 6.1 of !VDAF}}) | set by constructor      |
+  | `Field.MODULUS: int`| prime modulus defining the field arithmetic           | to be defined           |
+  {: #field-params title="Basic Field parameters."}
+
+
+Mastic uses Extendable Output Functions (XOFs) as specified in {{Section 6.2 of
+  !VDAF}}. The following XOF API functions referenced in this document are as
+  defined therein:
+
+  * `XofFixedKeyAes128(seed: bytes, dst: bytes, binder: bytes) -> Xof` is a XOF
+    based on a circular collision-resistant hash function from fixed-key AES that
+    is initialized with a given seed, a domain separation tag, and binder string,
+    and returns a `Xof` instance. The length of `seed` will typically be
+    SEED_SIZE, but some XOFs may support multiple sizes. The `seed` MUST be
+    generated securely (i.e., it is either the output of a CSPRNG or a previous
+    invocation of the XOF). The length of the domain separation string `dst`
+    passed to XofTurboShake128 MUST NOT exceed 65535 bytes.
+  * `XofTurboShake128(seed: bytes, dst: bytes, binder: bytes) -> Xof` is a XOF
+    wrapper for TurboSHAKE128 that is initialized with a given seed, a domain
+    separation tag, and binder string, and returns a `Xof` instance. The length of
+    `seed` will typically be SEED_SIZE, but some XOFs may support multiple sizes.
+    The `seed` MUST be generated securely (i.e., it is either the output of a
+    CSPRNG or a previous invocation of the XOF). The length of the domain
+    separation string `dst` passed to XofTurboShake128 MUST NOT exceed 65535
+    bytes.
+  * `Xof.next(length: int) -> bytes` returns the next `length` bytes of output of
+    a `Xof` instance.
+
+The following parameters are referenced in this document:
+
+| Parameter           | Description                                           | Value                                                      |
+|:--------------------|:------------------------------------------------------|:-----------------------------------------------------------|
+| `SEED_SIZE: int`    | the size (in bytes) of the XOR seed                   | 32 for `XofTurboShake128` or 16 for `XofFixedKeyAes128`    |
+{: #xof-params title="XOF parameters."}
+
+
+Mastic uses Fully Linear Proof (FLP) Systems as specified in {{Section 7.1 of
+  !VDAF}}. The following FLP API functions referenced in this document are as
+  defined therein:
+
+  * `flp.decide(verifier: list[F]) -> bool` is a deterministic decision algorithm
+    of a verifier that takes as input a verifier message and outputs a Boolean
```

Comment:
"algorithm of a verifier that takes as input a verifier message": does the first "verifier" refer to a party in the protocol or the verifier message? If you mean the former, then I think we need to define this; if you mean the latter, then this might grammatically problematic.